includes/db_connection.php

```diff
@@ -1,10 +1,14 @@
 <?php
-	// mysqli connection
-	//require_once('includes/settings.php');
```

Comment:
Do not include dead codes in comments. Simply remove the LOCs which are not required.